src/V3EmitCModel.cpp

```diff
@@ -405,6 +435,14 @@ class EmitCModel final : public EmitCFunc {
             puts("}\n");
         }
 
+        if (v3Global.timing()) {
+            putSectionDelimiter("Events and timing");
+            puts("bool " + topClassName()
+                 + "::eventsPending() { return !vlSymsp->TOP.__VdlySched.empty(); }\n");
+            puts("uint64_t " + topClassName()
+                 + "::nextTimeSlot() { return vlSymsp->TOP.__VdlySched.nextTimeSlot(); }\n");
```

Comment:
It will do as is for now. the __VdlySched should really live in the VlSyms not the TOP, but getting to it is harder that way, so we can do that later.